src/ctia/lib/map.clj

```diff
@@ -5,3 +5,12 @@
   (if (every? map? vals)
     (apply merge-with rmerge vals)
     (last vals)))
+
+(defn dissoc-in
+  [m [k & ks :as keys]]
+  (if ks
+    (if-let [nextmap (get m k)]
+      (let [newmap (dissoc-in nextmap ks)]
+        (assoc m k newmap))
+      m)
+    (dissoc m k)))
```

Comment:
I've just needed a `dissoc-in` today, I simply used `(update-in m [ks] :dissoc last-key)`